types/mongodb/index.d.ts

```diff
@@ -3963,7 +3963,7 @@ export class Cursor<T = Default> extends Readable {
      * @param value The flag boolean value.
      * @see https://mongodb.github.io/node-mongodb-native/3.6/api/Cursor.html#addCursorFlag
      */
-    addCursorFlag(flag: string, value: boolean): Cursor<T>;
+    addCursorFlag(flag: 'tailable' | 'oplogReplay' | 'noCursorTimeout' | 'awaitData' | 'partial' | string, value: boolean): Cursor<T>;
```

Comment:
According to the documentation it can _only_ be on of those:

https://mongodb.github.io/node-mongodb-native/3.6/api/Cursor.html#addCursorFlag

```suggestion
    addCursorFlag(flag: 'tailable' | 'oplogReplay' | 'noCursorTimeout' | 'awaitData' | 'partial', value: boolean): Cursor<T>;
```